Jenkinsfile

```diff
@@ -40,5 +40,5 @@
 // I.e. for testing library changes
 //@Library(value="pipeline-lib@your_branch") _
 
-packageBuildingPipeline(['distros' : ['centos7', 'centos8', 'leap15']])
+packageBuildingPipeline(['distros' : ['el8', 'leap15']])
```

Comment:
Can we remove centos7 here?   In 2.2, we still build and test on centos7

Reply:
I guess old packaging will still be there so maybe it's fine.

Reply:
That's what I thought, but someone correct me if this is wrong :)